### DISCUSSION TITLE
How to select font features?

### DISCUSSION THREAD

**Discussion by ghost** [rsms/inter#527] [Q&A]
I want to use Inter as my system font (on Windows and GNU/Linux), but I don't know how to toggle features. Should I build the font on my own, or change my system settings?

Thanks!

**Comment by alexleduc76** [comment]
Just don't use the variable font version. I use Inter Light (static version) on Linux and it works perfectly

**Comment by kepstin** [comment]
On Linux, it's possible to create a fontconfig file which will configure features in many apps. For example, I'm using this config file on one of my systems to set some defaults when using Inter as UI font:

(Install this to `~/.config/fontconfig/conf.d/20-inter-features.conf` for your user or `/etc/fonts/conf.d/20-inter-features.conf` for system-wide)
```xml
<?xml version="1.0"?>
<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
<fontconfig>
	<match target="font">
		<test qual="any" name="family"><string>Inter</string></test>
		<!-- These are set with "prepend" so that applications which configure features don't get overridden -->
		<edit name="fontfeatures" mode="prepend">
			<string>tnum</string><!-- Tabular numbers -->
			<string>ss03</string><!-- Curved r -->
			<string>ss04</string><!-- Disambiguation w/o zero -->
			<string>calt off</string><!-- Contextural alternatives -->
		</edit>
	</match>
</fontconfig>
```

A notable application which will not follow this configuration is Firefox - it seems to have native Opentype feature support, and ignores the feature settings that fontconfig suggests.
